tests/import_test.go

```diff
@@ -6,18 +6,44 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 
+	"k8s.io/api/core/v1"
+	
+	"kubevirt.io/containerized-data-importer/pkg/controller"
 	"kubevirt.io/containerized-data-importer/tests/framework"
 	"kubevirt.io/containerized-data-importer/tests/utils"
 )
 
 const (
 	testSuiteName           = "Importer Test Suite"
 	namespacePrefix         = "importer"
+	bedEndpoint             = "http://gopats.com/who-is-the goat.iso"
+	endpoint                = "http://distro.ibiblio.org/tinycorelinux/9.x/x86/release/Core-current.iso" 
+	annServiceAccountName   = "kubernetes.io/service-account.name"
+	secretName              = "mysecret"
 	waitForCDIImportTimeout = 30 * time.Second
 )
 
 var _ = Describe(testSuiteName, func() {
-	f := framework.NewFrameworkOrDie(namespacePrefix)
+	f, err := framework.NewFramework(namespacePrefix, framework.Config{})
```

Comment:
Consider calling the new [`NewFrameworkOrDie`](https://github.com/kubevirt/containerized-data-importer/blob/master/tests/framework/framework.go#L101). You do not need to check err and you do not need to pass in Config.